### ISSUE TITLE
guitarpro.models.GPException: reading measure 25, got ValueError: (1, 2) is not a valid KeySignature

### ISSUE THREAD

**Issue by kolanos** [Perlence/PyGuitarPro#28]
Getting the following error with [this guitar pro tab](https://tabs.ultimate-guitar.com/tab/kings-x/fade-guitar-pro-482085):

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../lib/python3.9/site-packages/guitarpro/io.py", line 59, in parse
    return gpfile.readSong()
  File ".../lib/python3.9/site-packages/guitarpro/gp5.py", line 80, in readSong
    self.readMeasures(song)
  File "/usr/local/Cellar/python@3.9/3.9.13_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/contextlib.py", line 137, in __exit__
    self.gen.throw(typ, value, traceback)
  File ".../lib/python3.9/site-packages/guitarpro/iobase.py", line 143, in annotateErrors
    raise gp.GPException(f"{action} {', '.join(location)}, "
guitarpro.models.GPException: reading measure 25, got ValueError: (1, 2) is not a valid KeySignature 
```

Version: 0.9

**Comment by Perlence** [comment]
Hey @kolanos, this one is interesting. They use B major since the beginning and mark measure 25 with C# minor. However, I would expect the key change to use the `(4, 1)` tuple, `4` being the number of sharps and `1` is for minor, but it uses `(1, 2)` instead. The meaning of this is not clear, however GP5 seems to parse it correctly.

![image](https://user-images.githubusercontent.com/1044300/175768959-7e73e9d9-781c-4ee5-bee6-4744143adc5b.png)

Perhaps, PyGuitarPro missed reading some bytes and it caused a misalignment, I'll have to look into it more.

**Comment by Perlence** [comment]
Fixed in dc6a01ebc9d43efe3cace48247df61315d5f0253 and I've published it in [v0.9.1](https://pypi.org/project/PyGuitarPro/0.9.1/). Thanks for filing the bug!
